# Automatically generated by Pynguin.
import blockchainetl_common.executors.retriable_value_error as module_0

def test_case_0():
    retriable_value_error_0 = module_0.RetriableValueError()